modules/storages/app/services/storages/one_drive_managed_folder_sync_service.rb

```diff
@@ -65,21 +65,30 @@ def call
 
     private
 
+    # rubocop:disable Metrics/AbcSize
     def apply_permission_to_folders
       info "Setting permissions to project folders"
       active_project_storages_scope.includes(:project).where.not(project_folder_id: nil).find_each do |project_storage|
```

Comment:
This is the main change, of how the interface is used.

Reply:
🟡 I'd keep the block variable as `origin_user_id` otherwise it can lead future devs to think it is a `User#id`.

Reply:
done